tasks/request_status_for_job/test/unit_tests/test_request_status_for_job.py

```diff
@@ -318,6 +321,6 @@ def test_task_output_json_schema(self, mock_get_user_connection: MagicMock):
 
         mock_get_user_connection.assert_called_once_with(db_connect_info)
         with open("schemas/output.json", "r") as raw_schema:
-            schema = json.loads(raw_schema.read())
+            json.loads(raw_schema.read())
```

Comment:
If schema isn't used, we shouldn't load it. I'd say revert this, comment out lines 323-325, and add a card to properly check the output of this lambda.

Reply:
commented out for now and created orca-519

Reply:
Took a closer look... Seems that these lines are duplicates of the initialization of _OUTPUT_SCHEMA. We can leave the card, since I know we want to add pydantic validation. These commented out lines can be removed.

Reply:
sounds good. Removed.